conftest.py

```diff
@@ -22,21 +22,20 @@ def pytest_addoption(parser):
                      help='Number of Jenkins masters to launch.')
     parser.addoption('--jobs', action='store', default=1, type=int,
                      help='Number of test jobs to launch.')
-    parser.addoption('--single-use', action='store', default=False,
-                     type=bool, help='Use Mesos Single-Use agents')
+    parser.addoption('--single-use', action='store_true', default=False,
```

Comment:
might not need default here, since if this flag is present it is always true, makes it a little confusing for user. (same in other places)